account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -12,7 +12,36 @@ class AccountMoveLine(models.Model):
         string="Is tax data editable?", compute="_compute_is_tax_editable"
     )
 
+    tax_line_id = fields.Many2one(inverse="_inverse_tax_line_id")
+
     @api.depends("move_id.state")
     def _compute_is_tax_editable(self):
         for rec in self:
             rec.is_tax_editable = rec.move_id.state == "draft"
+
+    def _inverse_tax_line_id(self):
+        for rec in self:
+            repartition_type = rec.tax_repartition_line_id.repartition_type or "tax"
+            factor_percent = rec.tax_repartition_line_id.factor_percent or 100
+            has_account = bool(rec.tax_repartition_line_id.account_id)
+            if rec.move_id.move_type in ("out_refund", "in_refund"):
```

Comment:
```suggestion
            if rec.move_id.type in ("out_refund", "in_refund"):
```

Do not fails because move_type is definde here, but with other sense:
https://github.com/OCA/account-financial-reporting/blob/d74eb8840f183f6f603603934fec9c71c84b1463/account_tax_balance/models/account_move.py#L21

Reply:
thanks